src/style.css

```diff
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  margin: 0;
+  --accent_colour: #00acdc;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+.logo_link {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.logo_img {
+  height: 40px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav_list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav_list_item_link {
+  height: 100%;
+  text-decoration: none;
+  color: black;
+  display: block;
+}
+
+.nav_list_item {
+  height: 60px;
```

Comment:
don't set element height directly, you can do it with elements inside

Reply:
I agree, in this case it was inappropriate